API-Reference/Data-Parallelism.md

```diff
@@ -30,7 +30,10 @@ Reducer objects used in conjunction with [parallel_reduce](Kokkos%3A%3Aparallel_
 
 ## Tags for Team Policy Calculations
 
-Tags passed to `team_size_max` or `team_size_recommended` to identify whether the pattern for which the team size for a functor is being requested
+Tags passed to helper functions `team_size_max` or
+`team_size_recommended` to identify the team size for the parallel pattern.  
```

Comment:
Maybe remove "whether" in the original version

Reply:
OK, I did a hard reset on my branch, and will start from scratch.  In the hard reset, `Kokkos::parallel_for.md` is unchanged, as is `Data-Parallelism.md.`.  I will work on the wording in `Data-Parallelism.md.`

Reply:
so what about:
Tags passed to helper functions `team_size_max` or `team_size_recommended`. Used to identify the pattern for which the `team_size_max` or `team_size_recommended` is calculated.

Reply:
> The sentence was meant to express what the Tags signify. They signify which parallel pattern will be used, since the recommended or max team size will be different depending on which pattern (`parallel_for`, `parallel_reduce`, `parallel_scan`).

I reworked the sentence.  See if you think the new version clearer & in line with what you need in the API-Reference.